tool_path_planner/src/utilities.cpp

```diff
@@ -298,4 +299,48 @@ bool createToolPathSegment(const pcl::PointCloud<pcl::PointNormal>& cloud_normal
   return true;
 }
 
+// this is a helper function for splitPaths()
+double getDistance(Eigen::Isometry3d t1, Eigen::Isometry3d t2)
+{
+  auto p1 = t1.translation();
+  auto p2 = t2.translation();
+  return sqrt(pow(p1.x() - p2.x(), 2) + pow(p1.y() - p2.y(), 2) + pow(p1.z() - p2.z(), 2));
+}
+
+ToolPaths splitSegments(ToolPaths tool_paths, double max_segment_length)
+{
```

Comment:
We need to have some mechanism to bypass splitSegments(). For example, we could set max_segment_length to 0 or even negative, if so, have this routine do nothing.

Reply:
Addressed in 12181b3